tests/test_runtime/test_apis.py

```diff
@@ -0,0 +1,35 @@
+import os
+from pathlib import Path
+
+import pytest
+
+from mmdet.apis import init_detector
+
+
+def test_init_detector():
+    project_dir = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
+    project_dir = os.path.join(project_dir, '..')
+
+    config_file = os.path.join(
+        project_dir, 'configs/mask_rcnn/mask_rcnn_r50_fpn_1x_coco.py')
+
+    # test init_detector with config_file: str
+    model = init_detector(config_file, device='cpu')
+
+    # test init_detector with config_file: str and cfg_options
+    cfg_options = dict(
+        model=dict(
+            backbone=dict(
+                depth=101,
+                init_cfg=dict(
+                    type='Pretrained', checkpoint='torchvision://resnet101'))))
+    model = init_detector(config_file, device='cpu', cfg_options=cfg_options)
```

Comment:
I think the below test also tests the case of config_file being str. So we can keep the below one and delete the above one to save unittest time.

We can also change the ResNet50 to ResNet18 in cfg_options.